src/modules/ticket/actionCreators.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  BuyTicketFailAction,
+  BuyTicketRequestAction,
+  BuyTicketSuccessAction,
+} from './reducer';
```

Comment:
Let's import and use as well the constant for the actions types from the reducer, since we have them 😄 
```suggestion
import {
  BUY_TICKET_FAIL,
  BUY_TICKET_REQUEST,
  BUY_TICKET_SUCCESS,
  BuyTicketFailAction,
  BuyTicketRequestAction,
  BuyTicketSuccessAction,
} from './reducer';
```